src/helpers/cli.js

```diff
@@ -1,9 +1,28 @@
 import yargs from "yargs";
+import { init as generate } from "./../generate";
+import { init as draft } from "./../draft";
+import { init as fetch } from "./../fetch";
 
 export const setUpCli = () =>
   yargs
-    .command(["generate", "$0"], "Generate release notes", commandLineSetUp)
-    .command("draft", "Create draft release on Github", commandLineSetUpPublish)
+    .command(
+      ["generate", "$0"],
+      "Generate release notes",
+      commandLineSetUp,
+      _ => generate(_)
```

Comment:
This should be able to be substituted from `_ => generate(_)` to `generate`, I think.